tests/testthat/test-ui-violin.R

```diff
@@ -63,8 +63,12 @@ test_that("cXviolin15", {
 
 test_that("cXviolin16", {
     check_ui_test(cXviolin16())
+
+    warning('Data points are appearing on the chart are too small, unless it is resized manually or refreshed')
```

Comment:
remove the first 'are' - ie should say 'Data points appearing on the chart are too small, unless it is resized manually or refreshed'